challenge_spr/src/challenge_spr_states/detect.py

```diff
@@ -57,56 +43,21 @@ def recognize(self, tries):
 
         for i in range(0, tries):
             self.robot.speech.speak(sentences[i % (tries - 1)], block=False)
-            image = self.get_shot()
-            face_rois = self.get_faces(image)
+            self.robot.head.look_at_point(VectorStamped(100, 0, 1.5, self.robot.robot_name + "/base_link"))
+            self.robot.head.wait_for_motion_done()
```

Comment:
You changed behavior, so this isn't a refactor?

Reply:
I didn't plan to change behavior. I see I forgot the sleep. Or did I forget more?